src/App.js

```diff
@@ -10,6 +10,7 @@ import {
 } from 'react-router-redux';
 import { Provider } from 'react-redux';
 import createSagaMiddleware from 'redux-saga';
+// eslint-disable-next-line import/no-extraneous-dependencies
 import { composeWithDevTools } from 'redux-devtools-extension/developmentOnly';
```

Comment:
Even though this was part of dependencies, the full version was probably not being included in the final bundle. Have you seen? [zalmoxisus/redux-devtools-extension#14-using-in-production](https://github.com/zalmoxisus/redux-devtools-extension#14-using-in-production)